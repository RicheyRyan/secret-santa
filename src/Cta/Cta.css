.cta {
  width: 100%;
  text-align: center;
  padding-bottom: 1.5em;
}

.cta p, .cta button{
  display: inline;
}

.cta p{
  margin-right: .4em;
}
