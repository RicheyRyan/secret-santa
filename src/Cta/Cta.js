import React from 'react';
import './Cta.css';

const Cta = props =>
  <div className="cta">
    <p>{props.message}</p>
    <button onClick={props.onClick}>
      {props.label}
    </button>
  </div>;

export default Cta;
