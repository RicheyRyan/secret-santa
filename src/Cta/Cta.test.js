import React from 'react';
import renderer from 'react-test-renderer';
import Cta from './Cta.js';

it('Cta component matches snapshot', () => {
  const component = renderer.create(
    <Cta label={'Test Label'} message={'Test Message'} onClick={() => {}} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
