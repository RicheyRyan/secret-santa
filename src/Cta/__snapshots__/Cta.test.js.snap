exports[`test Cta component matches snapshot 1`] = `
<div
  className="cta">
  <p>
    Test Message
  </p>
  <button
    onClick={[Function]}>
    Test Label
  </button>
</div>
`;
